README.md

```diff
@@ -18,29 +48,16 @@ xrootdredirector = cms-xrd-global.cern.ch
 
 ```
 
-In case you update module, you should reinstall the packages dependencies using:
-
-```sh
-pip install --upgrade .
+## Running the framework
+You can first perform a test run. Do:
 ```
-
-or use the bash script:
-```sh
-source recompile.sh
+ ./macros/steer.py -c -t
 ```
-
-Known changes needed:
+If everything looks good, you can submit condor jobs:
 ```
-     `sed -i 's/"fixedGridRhoFastjetAll"/"Rho_fixedGridRhoFastjetAll"/g' bamboo/bamboo/analysisutils.py`
-    
-     `sed -i 's/JRDatabase/jme-validation/g' bamboo/bamboo/analysisutils.py`
-    
-     `sed -i 's/JECDatabase/jme-validation/g' bamboo/bamboo/analysisutils.py`
-    
-     `sed -i '/cms-jet\/jme-validation/s/$/ branch="main",/' bamboo/bamboo/analysisutils.py`
-    
-     `sed -i 's/heads\/master"/heads\/"+self.branch/g' bamboovenv/lib/python3.9/site-packages/CMSJMECalculators/jetdatabasecache.py`
-    
-     `sed -i 's/idxs=defCache(self.rng)/idxs=defCache(self.rng).replace("IndexRange<std::size_t>{","IndexRange<std::size_t>{static_cast<unsigned long>(").replace("}",")}")/g' bamboo/bamboo/treeoperations.py`
-
+./macros/steer.py -c -s
+```
+Finally, you can make plots:
+```
+./macros/steer.py -p
 ```
```

Comment:
We could add something like in case you want to produce your own private NanoAOD files we have the JMENano framework here https://gitlab.cern.ch/cms-jetmet/jmenano

Reply:
do we want to add something like at the moment we have two modules DY for eff, purity and tau plots, QCD for MC truth or so?

Reply:
Agreed.